src/cable.cr

```diff
@@ -32,8 +31,12 @@ module Cable
     setting token : String = "token", example: "token"
     setting url : String = ENV.fetch("REDIS_URL", "redis://localhost:6379"), example: "redis://localhost:6379"
     setting disable_sec_websocket_protocol_header : Bool = false
-    setting backend_class : Cable::BackendCore.class = Cable::RedisBackend, example: "Cable::RedisBackend"
-    setting redis_ping_interval : Time::Span = 15.seconds
+    setting backend_class : Cable::BackendCore.class = Cable::RegistryBackend, example: "Cable::RedisBackend"
+    setting backend_ping_interval : Time::Span = 15.seconds
+    @[Deprecated("Use backend_ping_interval")]
```

Comment:
I'd be curious to see if this shows... Since the [method](https://github.com/luckyframework/habitat/blob/4d242eb524e00289fb5857ed1bd786ec96670a5d/src/habitat.cr#L245) doesn't take annotations in to consideration, this may not actually render. That would be a nice feature for Habitat though.